### ISSUE TITLE
Error with verification step

### ISSUE THREAD

**Issue by Brendan179** [mrworf/photoframe#99]
Hello,
I am running into an issue getting the photoframe to verify correctly. I have created credentials in google's API dashboard but when I upload that file, I am getting the following message. I have zero experience programming so I don't know what it is telling me to do differently. Any help would be greatly appreciated but I will continue to try.

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./frame.py", line 109, in wrap
    return fn(*args, **kwargs)
  File "./frame.py", line 517, in service_link
    return redirect(services.oauthStart(service))
  File "/root/photoframe/modules/servicemanager.py", line 182, in oauthStart
    return svc.startOAuth()
  File "/root/photoframe/services/base.py", line 239, in startOAuth
    return self._OAUTH.initiate()
  File "/root/photoframe/modules/oauth.py", line 106, in initiate
    self.rid = self.getRedirectId()
  File "/root/photoframe/modules/oauth.py", line 102, in getRedirectId
    r = requests.get('%s/?register' % self.ridURI)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 70, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 56, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 488, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 609, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/adapters.py", line 497, in send
    raise SSLError(e, request=request)
SSLError: ("bad handshake: SysCallError(104, 'ECONNRESET')",)

**Comment by mrworf** [comment]
I'll fix it within the next hour 

On July 7, 2019 10:30:42 AM PDT, Brendan179 <notifications@github.com> wrote:
>Hello,
>I am running into an issue getting the photoframe to verify correctly.
>I have created credentials in google's API dashboard but when I upload
>that file, I am getting the following message. I have zero experience
>programming so I don't know what it is telling me to do differently.
>Any help would be greatly appreciated but I will continue to try.
>
>Traceback (most recent call last):
>File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1612, in
>full_dispatch_request
>    rv = self.dispatch_request()
>File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1598, in
>dispatch_request
>    return self.view_functions[rule.endpoint](**req.view_args)
>  File "./frame.py", line 109, in wrap
>    return fn(*args, **kwargs)
>  File "./frame.py", line 517, in service_link
>    return redirect(services.oauthStart(service))
>File "/root/photoframe/modules/servicemanager.py", line 182, in
>oauthStart
>    return svc.startOAuth()
>  File "/root/photoframe/services/base.py", line 239, in startOAuth
>    return self._OAUTH.initiate()
>  File "/root/photoframe/modules/oauth.py", line 106, in initiate
>    self.rid = self.getRedirectId()
>  File "/root/photoframe/modules/oauth.py", line 102, in getRedirectId
>    r = requests.get('%s/?register' % self.ridURI)
>File "/usr/lib/python2.7/dist-packages/requests/api.py", line 70, in
>get
>    return request('get', url, params=params, **kwargs)
>File "/usr/lib/python2.7/dist-packages/requests/api.py", line 56, in
>request
>    return session.request(method=method, url=url, **kwargs)
>File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 488,
>in request
>    resp = self.send(prep, **send_kwargs)
>File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 609,
>in send
>    r = adapter.send(request, **kwargs)
>File "/usr/lib/python2.7/dist-packages/requests/adapters.py", line 497,
>in send
>    raise SSLError(e, request=request)
>SSLError: ("bad handshake: SysCallError(104, 'ECONNRESET')",)
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/mrworf/photoframe/issues/99

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by mrworf** [comment]
Hmm, try surfing to https://photoframe.sensenet.nu/?register=234234 from your web browser. That should return a string. Because your frame fails to connect to the redirect service needed during setup.

**Comment by Brendan179** [comment]
Thanks for the help! When I go there it gives me this: 

e1214081148ce93681a27087a326e5c0

**Comment by mrworf** [comment]
Hmm, so that works. Are you using the ready-made SD card image? Because the error would indicate that the frame has issues connecting to https://photoframe.sensenet.nu

Can you send the output from the log report button (and if that doesn't exists, you may need to force an update) 

On July 7, 2019 12:02:51 PM PDT, Brendan179 <notifications@github.com> wrote:
>Thanks for the help! When I go there it gives me this: 
>
>e1214081148ce93681a27087a326e5c0
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/mrworf/photoframe/issues/99#issuecomment-509022991

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by Brendan179** [comment]
[Log.pdf](https://github.com/mrworf/photoframe/files/3366039/Log.pdf)

Here is the log report. I should mention that I was able to get it to work a few days ago after struggling with the same problem but I don't know what I did to fix it then. I was trying to change the account over to another google account to give it as a gift when I ran into the same problem. I figured I was just doing something wrong. Thanks again
